include/mqt-core/dd/FunctionalityConstruction.hpp

```diff
@@ -26,6 +26,10 @@ MatrixDD buildFunctionality(const qc::Grover* qc, Package<Config>& dd);
 template <class Config>
 MatrixDD buildFunctionalityRecursive(const qc::Grover* qc, Package<Config>& dd);
 
+template <class Config>
+MatrixDD buildFunctionality(const QuantumComputation* qc, Package<Config>& dd,
+                            const std::optional<std::size_t> ncycles);
+
```

Comment:
```suggestion
```
I am fairly sure this function has been removed on the `main` branch and is no longer necessary.